nf_core/pipeline-template/pyproject.toml

```diff
@@ -1,10 +1,14 @@
-# Config file for Python. Mostly used to configure linting of bin/check_samplesheet.py with Black.
+# Config file for Python. Mostly used to configure linting of bin/check_samplesheet.py with Ruff.
 # Should be kept the same as nf-core/tools to avoid fighting with template synchronisation.
-[tool.black]
+[tool.ruff]
 line-length = 120
```

Comment:
for this to have an effect, I think we need to select also rule `E5` https://docs.astral.sh/ruff/rules/line-too-long/